source/contracts/libraries/token/BasicToken.sol

```diff
@@ -28,6 +28,18 @@ contract BasicToken is ERC20Basic {
     }
 
     /**
+    * @dev allows internal token transfers
+    * @param _from The source address
+    * @param _to The destination address 
+    */
+    function internalTransfer(address _from, address _to, uint256 _value) internal returns (bool) {
+        balances[_from] = balances[_from].sub(_value);
+        balances[_to] = balances[_to].add(_value);
+        Transfer(_from, _to, _value);
```

Comment:
`emitTransferLog` should be called after `Transfer` (assuming you don't rename it per CH issue).

Reply:
Would this in effect cause 2 Transfer events to happen?

Reply:
Yes, it will in some cases. One of the transfer events will be to maintain ERC20 spec, the other would be what we use in Augur Node for tracking transfers.

Reply:
Change made